generator.py

```diff
@@ -0,0 +1,35 @@
+from spike import Spike
+from random import randint
+
+
+class Generator:
+
+    def __init__(self):
+        self.spikes_config = []
+        self.spikes = []
+
+    def configure(self, config):
+        self.spikes = []
+        self.spikes_config = config
+        for _ in range(len(self.spikes_config)):
+            self.spikes.append([])
+
+    def generate(self, speed=7):
+        for i, config in enumerate(self.spikes_config):
+            if (len(self.spikes[i]) < config[3]):
+                side = int(randint(0, 1))
+                self.spikes[i].append(Spike(config[side][0],
+                                            config[side][1],
+                                            side, speed, config[2]))
+
+    def update(self, guys=[]):
```

Comment:
I'm curious, why the `guys` param?

Reply:
checks collision with each guys and returns True if game over, false otherwise

Reply:
Yes, I should have phrased that differently.

I meant we should change the variable to something more generic like sprite.

Reply:
So is Sprite name fine? Should I change this everywhere guy name is used

Reply:
Yes sprite is fine if you can't think of anything else, turtleart uses sprites that's why it came to my mind.